modules/apps/site/site-teams-web/src/main/resources/META-INF/resources/select_users.jsp

```diff
@@ -45,7 +45,7 @@ SelectUsersDisplayContext selectUsersDisplayContext = new SelectUsersDisplayCont
 					%>
 
 					<liferay-ui:search-container-column-text>
-						<clay:user-card-v2
+						<clay:user-card
```

Comment:
Couldn't wait to do this? 😂 

Just remember these are scheduled for different tasks so we can send to the teams for validation and testing. Let's make sure we don't push this inadvertently.